packages/tools/client-debugger/package.json

```diff
@@ -1,102 +1,102 @@
 {
-  "name": "@fluid-tools/client-debugger",
```

Comment:
Looks like a lot of unneeded formatting changes here. Are these correcting prettier violations? If not, we should probably revert.

Reply:
Changes in `package.json` is irrelevant to the task. However, `npm run build` did not run unless I applied formatting changes to the file. I could revert the changes for this PR and create a separate PR to enforce formatting in `client-debugger` and `client-debug-view` directories.

Reply:
Okay, let's leave the changes then. Thanks!